consensus/scp/play/src/main.rs

```diff
@@ -63,8 +57,12 @@ fn trivial_validity_fn(_value: &TxHash) -> Result<(), TransactionValidationError
     Ok(())
 }
 
-fn trivial_combine_fn(values: BTreeSet<TxHash>) -> BTreeSet<TxHash> {
-    BTreeSet::from_iter(values.into_iter().take(MAX_TRANSACTIONS_PER_BLOCK))
+fn trivial_combine_fn(values: &[TxHash]) -> Vec<TxHash> {
```

Comment:
Does this need to be different from the `trivial_combine_fn` in [src/test_utils#L26](https://github.com/mobilecoinofficial/mobilecoin/blob/88f850c80ac70a7e1f9e8b49cd511ddb099eb085/consensus/scp/src/test_utils.rs#L26)?